### ISSUE TITLE
More text fonts

### ISSUE THREAD

**Issue by SirusCodes** [SirusCodes/Heartry#8]
**Describe the feature**
We can use google fonts to get fonts dynamically.

**Plan to execute**
Add a list of fonts(few selected ones) in the UI and change it when the user selects the font and updates the UI.

**Comment by Roshani9731** [comment]
Hy, I want contribute in this issue can you help me?

**Comment by SirusCodes** [comment]
Hey, @Roshani9731 I hope you already took a look at how the app works.
So we have [image_text_handler](https://github.com/SirusCodes/Heartry/blob/main/lib/screens/image_screen/widgets/image_text_hander.dart) to which is a bottom sheet for the main screen. First, you have to make a list of fonts available to change in a horizontal fashion(like Instagram) but in our app, it will be present in this bottom sheet.

Then you have to manage the font family using riverpod and update the text font as it is changed in [poem_image_text](https://github.com/SirusCodes/Heartry/blob/main/lib/screens/image_screen/widgets/poem_image_text.dart).

You can load fonts dynamically from [google_fonts](https://pub.dev/packages/google_fonts#:~:text=Or%2C%20if%20you%20want%20to%20load%20the%20font%20dynamically%3A) package.

Also after making a fork make a separate branch from [stable](https://github.com/SirusCodes/Heartry/tree/stable) branch

Do let me know if you have any questions.

**Comment by Sarahkhan20** [comment]
It is a Good Idea to implement Other fonts and also Bold Italic like a text editor maybe so that users don't have to use multiple apps for this and also some background options

**Comment by SirusCodes** [comment]
Thanks for the suggestion, @Sarahkhan20 can you suggest some good fonts?

**Comment by Sarahkhan20** [comment]
Yeah Sure! Why Not,  Some fonts that can be implemented in a poetry app can be  cursive fonts maybe , then sans-serif, serif fonts. You can use Times New Roman, Georgia which Gives modern poetry look as well they are readable. Can Implement Script fonts. What's the issue with using Google font APIs? I can send you a list if you want let me know! I'll be happy to help!

**Comment by uveshmenpur** [comment]
@SirusCodes  I want to work on this issue can you please assign to me

**Comment by SirusCodes** [comment]
> @SirusCodes  I want to work on this issue can you please assign to me

Sure done 👍

**Comment by uveshm003** [comment]
@SirusCodes  please assign this issue to me

**Comment by SirusCodes** [comment]
Released in 2.5
